File: src/main/java/domain/Trip.java
package domain;

import java.sql.Time;

public class Trip implements Identifiable<Integer> {
    private int id;
    private final String touristAttraction;
    private final String transportCompany;
    private final Time departureTime;
    private final float price;
    private final int seats;

    public Trip(String touristAttraction, String transportCompany, Time departureTime, float price, int seats) {
        this.touristAttraction = touristAttraction;
        this.transportCompany = transportCompany;
        this.departureTime = departureTime;
        this.price = price;
        this.seats = seats;
    }

    public Trip(int id, String touristAttraction, String transportCompany, Time departureTime, float price, int seats) {
        this.id = id;
        this.touristAttraction = touristAttraction;
        this.transportCompany = transportCompany;
        this.departureTime = departureTime;
        this.price = price;
        this.seats = seats;
    }


    public String getTouristAttraction() {
        return touristAttraction;
    }

    public String getTransportCompany() {
        return transportCompany;
    }

    public Time getDepartureTime() {
        return departureTime;
    }

    public float getPrice() {
        return price;
    }

    public int getSeats() {
        return seats;
    }

    @Override
    public Integer getId() {
        return id;
    }

    @Override
    public void setId(Integer id) {
        this.id = id;
    }
}

File: src/domain/Trip.cs
using System;

namespace AgentiiDeTurism.src.domain
{
    public class Trip : IIdentifiable<int>
    {
        public int Id { get; set; }
        public string TouristAttraction { get; set; }
        public string TransportCompany { get; set; }
        public TimeSpan DepartureTime { get; set; }
        public float Price { get; set; }
        public int Seats { get; set; }

        public Trip(string touristAttraction, string transportCompany, TimeSpan departureTime, float price, int seats)
        {
            this.TouristAttraction = touristAttraction;
            this.TransportCompany = transportCompany;
            this.DepartureTime = departureTime;
            this.Price = price;
            this.Seats = seats;
        }
        public Trip(int id, string touristAttraction, string transportCompany, TimeSpan departureTime, float price, int seats)
        {
            this.Id = id;
            this.TouristAttraction = touristAttraction;
            this.TransportCompany = transportCompany;
            this.DepartureTime = departureTime;
            this.Price = price;
            this.Seats = seats;
        }

        public int getId()  
        {
            return this.Id;
        }

        public void setId(int id)
        {
            this.Id = id;
        }

        public override string ToString()
        {
            return this.TouristAttraction + " at " + this.DepartureTime;
        }
    }
}
